data/configurables/ideology_mappings.txt

```diff
@@ -0,0 +1,34 @@
+// governance principles
+link = { law = law_chiefdom democratic = -25 communism = -50 fascism = -25 neutrality = 25 }
```

Comment:
If it were me, I'd reformat this file to simplify it:
law_chiefdom = { democratic = -25 communism = -50 fascism = -25 neutrality = 25 }

Reply:
Yes, good call.